README.md

```diff
@@ -11,9 +12,11 @@ It's built on top of the Bitcoin's blockchain and [IPFS](https://ipfs.io/).
 
 - [How to Run the Po.et Node](#how-to-run-the-poet-node)
     - [Install](#install)
+    - [Docker Compose](#docker-compose)
+    - [Makefile](#Makefile)
```

Comment:
This link doesn't work, at least in the preview. I think you need to change it to lower case.